lua/terrortown/lang/zh_hans.lua

```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
 L.slot_weapon_class = "职业槽"
 
 -- 2023-10-04
---L.label_voice_duck_spectator = "Duck spectator voices"
---L.label_voice_duck_spectator_amount = "Spectator voice duck amount"
---L.label_voice_scaling = "Voice Volume Scaling Mode"
---L.label_voice_scaling_mode_linear = "Linear"
---L.label_voice_scaling_mode_power4 = "Power 4"
---L.label_voice_scaling_mode_log = "Logarithmic"
+L.label_voice_duck_spectator = "降低观察者语音"
+L.label_voice_duck_spectator_amount = "降低观察者语音音量"
+L.label_voice_scaling = "语音音量缩放模式"
```

Comment:
```suggestion
L.label_voice_scaling = "语音淡化比例"
```
Literally "voice ducking scale".